test/unit/plugins/rest_spec.js

```diff
@@ -2918,4 +3049,71 @@ describe("The Rest plugin", function () {
 			});
 		});
 	});
+
+	describe("fails to create a Database for Cache", function () {
```

Comment:
Since we're using memory now, references to databases here should probably be removed.